plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_pt_BR.properties

```diff
@@ -67,7 +67,7 @@ dataTransfer.processor.sql.property.includeAutoGenerated.description
 dataTransfer.processor.sql.property.includeAutoGenerated.label                  = Incluir colunas geradas
 dataTransfer.processor.sql.property.omitSchema.description                      = Omita o nome do esquema / cat\u00E1logo nas instru\u00E7\u00F5es INSERT
 dataTransfer.processor.sql.property.omitSchema.name                             = Omit schema name
```

Comment:
`dataTransfer.processor.sql.property.omitSchema.name = Omit schema name`

Hello. Could you translate this also?